src/common/openapi.yaml

```diff
@@ -41,5 +41,7 @@ paths:
     $ref: 'paths/common@places.yaml'
   /common/currency/list:
     $ref: 'paths/common@currencies.yaml'
+  /common/currency/exchange_rate:
```

Comment:
it should not be `/common/currency/exchange_rate:` it should be `/common/exchange_rate:`

Reply:
I think we should keep **/currency/** folder in between for better separation/understanding

Also list of currencies is already implemented with that only, even places too.
cc: @aditya999123

Reply:
currency and exchange_rates both are different top level resources so should have different path after <role>/

Reply:
At line number 29 of this file, exchange rates have been mentioned under currencies.
(How come exchange rates and currency both are different? The exchange rate is of currencies only)

Reply:
Also get_exchange_rate() in platform-api is under api > currencies > common.py
and the currency_layer.py under core > services has both list currency and exchange_rates in same file